packages/web/src/components/PasswordField/_PasswordField.scss

```diff
@@ -0,0 +1,7 @@
+@use 'theme';
+@use '../../tools/forms';
+
+$-type: 'PasswordField';
```

Comment:
I would prefer `$_` for a local variable, but does it even have to be here? We can move it to the theme as well, and keep it there as just `$type`. It is similar to button variants and we keep them in the theme as well.

Reply:
Same for TextField please.

Reply:
I don't like the idea of having $type in tokens. It is not something you want to change (or allow anybody to change). I will hardcode it into mixin call.

Variants go to tokens, it makes sense 👍.